DependencyInjection/Compiler/FormFactoryCompilerPass.php

```diff
@@ -11,10 +11,10 @@
 
 namespace Sonata\CoreBundle\DependencyInjection\Compiler;
 
-use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\FormPass;
```

Comment:
There is no bundle with no requirement on fwb... That's why there argument can be "replaced" (they come from fwb already)